v21.2/restoring-backups-across-versions.md

```diff
@@ -0,0 +1,64 @@
+
+## Support for restoring backups into a newer version
+
+Cockroach Labs supports restoring a backup on one major version following the major version on which you took the backup. Therefore, if you took a backup on major version N, you can restore it on major version N+1.
+
+For example, backups taken on v21.1 can be restored into v21.2. Backups taken on v21.2 can be restored into v22.1. The following table outlines this support:
+
+Backup taken on version | Restorable into version
+------------------------+--------------------------
+20.1                    | 20.2
+20.2                    | 21.1
+21.1                    | 21.2
+21.2                    | 22.1
+22.1                    | 22.2
+
+See the following section, [Restore into a newer version](#restore-into-a-newer-version), for an example of of restoring from multiple major versions previous to a newer version.
+
+{{site.data.alerts.callout_info}}
+Cockroach Labs does **not** support restoring backups from a newer version into a previous version. 
+{{site.data.alerts.end}}
+
+## Restore into a newer version
+
+When you upgrade from version N to N+1, you can restore version N backup into version N+1. Then, you can take a new backup on the version you just upgraded to. If you need to restore a backup that was taken with on an earlier version, you would repeat this process until you have fully upgraded to the desired version.
+
+For example, you have taken a backup on v20.2 and you want to upgrade to v22.1. You would upgrade and back up through the following process:
+
+1. Upgrade from v20.2 to v21.1:
+    1. Finalize your upgrade. See [Upgrade to CockroachDB v21.1](../v21.1/upgrade-cockroach-version.html) for the necessary steps to upgrade.
+    1. [Restore your backup](restore.html#examples) from v20.2 into the v21.1 upgraded cluster.
+    1. [Take a new backup](backup.html#examples) on the v21.1 cluster.
+1. Upgrade from v21.1 to v21.2:
+    1. Finalize your upgrade. See [Upgrade to CockroachDB v21.2](../v21.2/upgrade-cockroach-version.html) for the necessary steps to upgrade.
+    1. Restore your backup from v21.1 into the v21.2 upgraded cluster.
+    1. Take a new backup on the v21.2 cluster.
+1. Upgrade from v21.2 to v22.1:
+    1. Finalize your upgrade. See [Upgrade to CockroachDB v22.1](../v22.1/upgrade-cockroach-version.html) for the necessary steps to upgrade.
+    1. Restore your backup from v21.2 into the v22.1 upgraded cluster.
+    1. Take a new backup on the v22.1 cluster.
+
+See the [Upgrade Policy](../cockroachcloud/upgrade-policy.html) page for details on version upgrades to {{ site.data.products.db }} clusters.
+
+## Support for long-term backup archival
+
+When you need to archive a backup for the long term, we recommend that you also archive the CockroachDB binary of the version that the backup was taken on. However, for this method, it is necessary that the binary can still be run **and** is able to upgrade to a version that is supported.
```

Comment:
Hmm, Im not sure this second part is necessary?

Reply:
I am assuming you mean from "However,..." onward? If so, this was something that was part of the original doc and it seemed important to mention as a consideration?

Reply:
I'm also not really sure what the However... here is saying. Like, you can use the archived binary and restore regardless of whether or not you then upgrade, so that seems unrelated?

Reply:
Yes, the "And is unable to upgrade" is not required.